my-react-app/src/components/LoginPage.js

```diff
@@ -0,0 +1,39 @@
+import { Auth } from 'aws-amplify';
+import React, { useState } from 'react';
+import './LoginPage.css';
+
+const LoginPage = () => {
+  const [username, setUsername] = useState('');
+  const [password, setPassword] = useState('');
+  const [error, setError] = useState('');
+
+  const handleLogin = async () => {
+    try {
+      await Auth.signIn(username, password);
+    } catch (err) {
+      setError(err.message);
```

Comment:
You might want to add basic validation to check if the username and password fields are not empty before attempting to sign in, and add some descriptive messages.